src/coupling/hm_iterative.cc

```diff
@@ -113,6 +133,12 @@ void HM_Iterative::EqData::initialize(Mesh &mesh)
     old_iter_pressure_ptr_ = create_field_fe<3, FieldValue<3>::Scalar>(beta_ptr_->get_dofhandler());
     div_u_ptr_ = create_field_fe<3, FieldValue<3>::Scalar>(beta_ptr_->get_dofhandler());
     old_div_u_ptr_ = create_field_fe<3, FieldValue<3>::Scalar>(beta_ptr_->get_dofhandler());
+
+
+    /// Initialize conducitvity ptr
+
+    conductivity_model_ptr_ = create(fn_K_mechanics(), mechanics_->data().cross_section, mechanics_->data().conductivity_k0, deta_.delta, data_.delta_min); 
```

Comment:
Somewhere should be used the output_cross_section field which depends on the displacement jump.